images/win/scripts/Tests/NodeLTS.Tests.ps1

```diff
@@ -0,0 +1,13 @@
+Describe "Node LTS" {
```

Comment:
Let's name file `Node.Tests.ps1` and Describe `Node.JS`.
I don't think that we should include `LTS` to test name.